### ISSUE TITLE
Starting and Stopping AudioKit has effect on AKAppleSampler

### ISSUE THREAD

**Issue by mlostekk** [AudioKit/AudioKit#1482]
I am having an issue when using the `AKAppleSampler` and starting and stopping the audioKit. Every time its stopped and started a lowpass filter is applied.

Here is the repro-repo

https://github.com/mlostekk/AudioKitIssue

To reproduce it do the following

1. Press Setup
2. Press Audiokit start
3. Press Play (sounds normal)
4. Press AudioKit stop
5. Press AudioKit start
6. Press Play (lowpassfilter gets applied a bit)
Now repeat 4, 5 and 6 and see that the LP filter is applied more with each start stop cycle

So basically this is showing that when I `AuidioKit.start()` and `AudioKit.stop()` it (_e.g. to modify the audio graph_), a LP is getting applied to the sampler. 

Any solution to this? Is this a know issue?

**Comment by aure** [comment]
How are you deciding that its a  low pass filter and not some other artifact of the stop/start process?  I'll have a look a bit later, just curious as to how you came to that conclusion.

**Comment by aure** [comment]
I actually don't think this sounds like a low pass filter, but more like there are more and more samplers playing simultaneously.  The sound becomes louder and more distorted as you continue.  Since AKAppleSampler is a pretty lightweight wrapper around Apple's sampler, I wouldn't be surprised if its an issue we can't resolve.

**Comment by aure** [comment]
As it is, I would advise to work around this.  Either run set up automatically when you restart AudioKit or never stop AudioKit, just mute it.  I'm not sure what else I can do so I'll close this for now.

**Comment by mlostek** [comment]
Oh okay. Well, can when i modify the graph while its not stopped? It seems to crash with this issue when i try to modify the graph when its not stopped, and then trigger an AKAudioPlayer to play.

https://github.com/AudioKit/AudioKit/issues/1469

**Comment by mlostek** [comment]
Sounds also a bit related to https://github.com/AudioKit/AudioKit/issues/1110 doesnt it?

**Comment by aure** [comment]
I think its a bit different than that issue in that this one is unlikely due to sample rate. Stop AudioKit and make sure to set up the sampler each time.  Should work alright I'd think?
